### PR TITLE
Made logo clickable to go to the home page

### PR DESCRIPTION
**Pull request by stevenyuser**
### Summary 

Made the QMI logo clickable, so that when it is clicked, it redirects the user to the "/home" page.

image

image

### Test Plan 

Tested clicking the logo on different screen sizes (mobile, computer, tablet).

### Notes 

Made the logo have an underline on hover. Also changed the user profile underline hover behavior to be consistent in style with the logo hover style.

### Breaking Changes 

None

- [ ] I have updated the documentation accordingly.
- [ ] My PR adds a @ts-ignore

### CODE REVIEW

File: src/styles/TopBar.scss [hunk 1]
```diff
@@ -57,12 +57,29 @@
             justify-content: center;
         }
 
-        .QMILogo {
+        // .logo {
```
Comment (rgu0114) [inline src/styles/TopBar.scss:60]: make sure to remove all comments before pushing

**Review by rgu0114 — CHANGES_REQUESTED** [review]
This looks great Steven! Thanks again for taking this on and for making sure to test on tablet, mobile, etc. We may want to check with design/product on the color of the underline, and make sure to remove commented out code. I'll approve as soon as those two things are resolved.

**Comment by stevenyuser** [comment]
Thanks for the feedback, Richard! I just removed the commented-out code.

**Review by burninglilies — APPROVED** [review]
Thanks for the changes Steven, I double checked the hover coloring for the logo shadow and it's consistent with the user profile hover. So looks good!

**Review by rgu0114 — APPROVED** [review]

**Pull request merged**